step-release-vis/src/app/components/environment/environment_test.ts

```diff
@@ -410,15 +413,52 @@ describe('EnvironmentComponent', () => {
 
   describe('time range drag', () => {
     it('should update time range', () => {
-      // TODO(#277): implement test
+      const svg = fixture.debugElement.query(
+        By.css(`#${component.environment.name}-svg`)
+      );
+      spyOn(component.startTimestampChange, 'emit');
+      spyOn(component.endTimestampChange, 'emit');
+      svg.triggerEventHandler('mousedown', {pageX: 100, pageY: 100});
+      fixture.detectChanges();
```

Comment:
Could you add this to a separate test, so we don't have multiple expect phases in a single test?
Also please add a blank line between setup/action/expect (as you did below)

Reply:
Done